src/identity/types.ts

```diff
@@ -141,6 +141,7 @@ export type IdentityContextType = {
   address: Address; // The Ethereum address to fetch the avatar and name for.
   schemaId?: Address | null; // The Ethereum address of the schema to use for EAS attestation.
   setAddress: Dispatch<SetStateAction<Address>>;
+  chain?: Chain; // Optional chain for domain resolution
```

Comment:
After this PR we need to update all docs with the latest code you wrote. And make sure Types are up to date, and also new functionality are there.

Reply:
Oh keep always key value in alphabetical order.

cc @roushou do you think we can lint this in CI? I am kind of tired to always mention this over code reviews.

Reply:
Sounds good, next PR will be for the docs.

Reply:
@Zizzamia Using Eslint yes https://eslint.org/docs/latest/rules/sort-keys.

I just had a look at Biome rules and don't see an equivalent :grimacing: https://github.com/biomejs/biome/discussions/3

Will make a PR tomorrow with Eslint and see how it goes. I think it would be fine to just have it run on CI.